docs/installation.md

```diff
@@ -0,0 +1,44 @@
+# Installing Pop!_OS
+
+The Pop!_OS installer guides the user through setup of basic system settings:
+
+- Locale and keyboard language
+- Hard drive selection and partitioning (advanced)
+- User account creation
+- System disk encryption
+
+## Installation
+
+1. Choose a language. 
+
+![Choose Language](images/installation/choose-language.png)
+
+2. Select a keyboard language layout.
+
+![Keyboard Language](images/installation/keyboard-language.png)
+
+3. Choose a clean or custom installation. <!-- See Using [GParted Custom (Advanced)](advanced-installation.md) options. -->
+
+![Clean Install](images/installation/clean-install.png)
+
+4. Choose the target drive for the OS installation. Click Erase and Install.
+
+![Select Drive](images/installation/select-system-drive.png)
+
+5. Enter a username.
+
+![Enter Username](images/installation/enter-username.png)
+
+6. Enter the account password.
+
+![Enter Password](images/installation/enter-password.png)
+
+7. Choose to encrypt the drive, and optionally create a separate encryption password.
+
+![Encrypt Drive](images/installation/encrypt-drive.png)
+
+> Note: Encrypting the system disk ensures user data is secure in the event that access is lost to the physical device. 
```

Comment:
> in the event that access is lost to the physical device

is a weird way to phrase this. The security threat isn't that I lost my device, it's that someone else has my device.

Reply:
changed phrasing